### PR TITLE
add groovy DSL support to classpath

### PR DESCRIPTION
**Pull request by htreu**
Signed-off-by: Henning Treu <henning.treu@telekom.de>

### CODE REVIEW

File: targetplatform/SmartHome-Runtime.launch [hunk 1]
```diff
@@ -24,7 +24,7 @@
...
...ault,org.eclipse.jetty.osgi.boot@default:true,org.eclipse.jetty.osgi.httpservice@default:default,org.eclipse.jetty.proxy@default:default,org.eclipse.jetty.security*9.2.9.v20150224@default:default,org.eclipse.jetty.server*9.2.9.v20150224@default:true,org.eclipse.jetty.servlet*9.2.9.v20150224@default:true,org.eclipse.jetty.util*9.2.9.v20150224@default:default,org.eclipse.jetty.webapp@default:default,org.eclipse.jetty.xml@default:default,org.eclipse.osgi.services@default:default,org.eclipse.osgi@-1:true,org.eclipse.text@default:default,org.eclipse.xtend.lib.macro@default:default,org.eclipse.xtend.lib@default:default,org.eclipse.xtext.common.types@default:true,org.eclipse.xtext.ecore@default:default,org.eclipse.xtext.smap@default:default,org.eclipse.xtext.util@default:default,org.eclipse.xtext.xbase.lib@default:default,org.eclipse.xtext.xbase@default:default,org.eclipse.xtext@default:default,org.json@default:default,org.jupnp@default:false,org.mapdb.mapdb@default:default,org.objectweb.asm@default:default,org.sat4j.core@default:default,org.sat4j.pb@default:default,org.slf4j.api@default:default,org.slf4j.jcl@default:default,org.slf4j.jul@default:default,org.slf4j.log4j@default:default"/>
```
Comment (kaikreuzer) [inline targetplatform/SmartHome-Runtime.launch]: What has changed in the launch config? I would assume that this is unrelated to this PR?

File: bundles/core/org.eclipse.smarthome.core.audio.test/.settings/org.eclipse.jdt.groovy.core.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+groovy.compiler.level=24
```
Comment (maggu2810) [inline bundles/core/org.eclipse.smarthome.core.audio.test/.settings/org.eclipse.jdt.groovy.core.prefs]: Shouldn't we use -1 as version, so the workspace one is used and not a different one if we ever want to change that.
We already fixed a bunch of references after the 23 to 24 bump.

Comment (htreu) [inline bundles/core/org.eclipse.smarthome.core.audio.test/.settings/org.eclipse.jdt.groovy.core.prefs]: I will add another PR, tnx.
